### DISCUSSION TITLE
No rule to make target

### DISCUSSION THREAD

**Discussion by mabannach** [abhiTronix/raspberry-pi-cross-compilers#121] [General]
Howdy,

trying to build the toolchain from scratch. Followed the instructions [for build scripts (here)](https://github.com/abhiTronix/raspberry-pi-cross-compilers/tree/master/build-scripts) and worked all good to step 3.

Both  `./RTBuilder_32b -g "8.3.0" -r "3+" -o "buster"` and `./RTBuilder_64b -g "14.2.0" -o "buster"` end up with 

```
...
config.status: executing default commands
make[1]: *** No rule to make target 'c,c++,fortran', needed by 'native'.  Stop.
make[1]: *** Waiting for unfinished jobs....
make: *** [Makefile:4284: all-gcc] Error 2
```

The builds have been on a clean debian/buster with fresh checkouts of the current (as of today) repo.

I must be missing something!?

Few other observations:
- rsync should be included in the tools to install (apt-get install -y rsync), I put in a pull request for that once I got everything working.
- The 8.3.0 gcc tries to compile `libsanitizer/asan/asan_linux.cpp`, but that version appears to have `libsanitizer/asan/asan_linux.cc`

**Comment by abhiTronix** [comment]
> The builds have been on a clean debian/buster with fresh checkouts of the current (as of today) repo.

@mabannach What OS and machine are you using? This script requires a x86-64 Intel/AMD machine with any Linux x86-64 (not arm) distro to run these scripts.
